src/python/grpcio/grpc/__init__.py

```diff
@@ -649,6 +649,10 @@ def unary_unary(
 
     Args:
       method: The name of the RPC method.
+      request_serializer: Optional method for serializing the request
```

Comment:
Let's use the more general term "behavior" rather than "method"; a method is not a function and a function is not a method but both are behaviors.